_includes/primary-nav.php

```diff
@@ -182,15 +182,18 @@ function toggleSearchBar () {
     });
 
     var subNavigation = $("#sub-navigation");
+    var deskNavigation = $("#desktop-navigation");
     var navHeight = subNavigation.offset().top;
 
     $(window).on("scroll", function () {
       if ($(window).scrollTop() > navHeight) {
         subNavigation.addClass("out");
+        deskNavigation.addClass("padded");
```

Comment:
@j2trumpet Can you explain what this class does? Is there any way to add this margin to the sub navigation div instead of the desktop? The JS component that monitors the scrolling is part of the sub nav.